docs/section10/10-14.md

```diff
@@ -0,0 +1,159 @@
+            - Else, set the was_heat_capacity_not_defined boolean variable equal to TRUE: `Else: was_heat_capacity_not_defined = TRUE`  
+            - Check if capacity is <225,000 Btuh (below checks if the modeled value is equal to the most stringent size category which is <225,000 for when capacity is not defined) or (if the aggregation factor was defined (i.e., does not equal Null) and the resultant capacity is less than 225000) or if the aggregation factor = Null (most strigent size category) : `if total_heat_capacity_b < 225000 OR was_heat_capacity_not_defined boolean variable == TRUE OR (zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 225000) OR zone_list_b[0].aggregation_factor == Null:`   
+                - Check if the capacity is greater than or equal to 225000 and zone aggregation not defined: `if total_heat_capacity_b >= 225000 AND zone_list_b[0].aggregation_factor == Null :`   
+                  - Zone aggregation factor needed so set boolean to true: `zone_aggregation_factor_undefined_and_needed = TRUE`    
+                - Else, set the boolean variable to FALSE: `Else: zone_aggregation_factor_undefined_and_needed = FALSE`  
+                - Loop through each of the efficiency_metric_types associated with the heating system until AFUE is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`  
+                    - Check if the efficiency metric equals ANNUAL_FUEL_UTILIZATION_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "ANNUAL_FUEL_UTILIZATION_EFFICIENCY":`  
+                        - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
+                        - Break out of loop if efficiency is found: `break`  
+                - Check that modeled_efficiency_b does not equal Null: `if modeled_efficiency_b != Null:`     
+                    - Set expected_baseline_eff_b equal to 78% AFUE: `expected_baseline_eff_b = 0.78`   
+                - Check if the modeled efficiency does not equal Null, if false (i.e., does not equal Null) then go to the Rule Assertion, if it equals Null perform logic below: `if modeled_efficiency_b == Null:`   
+                    - Loop through each of the efficiency_metric_types associated with the heating system until THERMAL_EFFICIENCY is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`  
+                        - Check if the efficiency metric equals THERMAL_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "THERMAL_EFFICIENCY":`  
```

Comment:
What if a project reports both `ANNUAL_FUEL_UTILIZATION_EFFICIENCY` AND `THERMAL_EFFICIENCY`, `AFUE=0.77` and `THERMAL_EFFICIENCY=0.81`?

Reply:
I don't know. I guess we figured that would not happen. Are you suggesting it should check both and then fail if either one is incorrect?

Reply:
Yes I think it should at least be one of these options you mention, instead of the current method of prioritizing AFUE

Reply:
Done